core/src/main/scala/kafka/coordinator/group/GroupMetadata.scala

```diff
@@ -112,6 +112,11 @@ private object GroupMetadata {
       Stable -> Set(CompletingRebalance),
       PreparingRebalance -> Set(Stable, CompletingRebalance, Empty),
       Empty -> Set(PreparingRebalance))
+  val unknownMemberMetadata = new MemberMetadata("", "", "", "", 0, 0, "", List[(String, Array[Byte])]())
+  def isUnknownMember(member: MemberMetadata): Boolean = member == unknownMemberMetadata
+
+  // TODO: decide if ids should be predictable or random
```

Comment:
nit: since we're touching this, how about we get rid of this TODO?

Reply:
I guess you mean we should just remove it?